elasticsearch/src/test/scala/akka/stream/alpakka/elasticsearch/ElasticsearchSpec.scala

```diff
@@ -720,11 +721,11 @@ class ElasticsearchSpec extends WordSpec with Matchers with BeforeAndAfterAll {
   }
 
   "ElasticsearchSource" should {
-    "should prefetch next scroll" in {
+    "should let you search without specifying typeName" in {
       val f1 = ElasticsearchSource
         .typed[Book](
           indexName = "source",
-          typeName = "book",
+          typeName = None,
           query = """{"match_all": {}}""",
           settings = ElasticsearchSourceSettings(bufferSize = 5)
         )
```

Comment:
Maybe add a test instead of changing an existing one?

Reply:
I've also renamed the test.

The "old test" was added by me in my previous PR.

I used it to verify that prefetching of next scroll was working, but as you see, it does not assert anything other than that it receives all docs.

So since it did not actually assert anything special about "should prefetch next scroll", I just re-purposed it instead of removing it and adding a new one..

Reply:
Ok, thanks for clarifying!